src/broker/overlay.c

```diff
@@ -426,6 +427,15 @@ static struct child *child_lookup (struct overlay *ov, const char *id)
     return NULL;
 }
 
+bool overlay_msg_is_local (const flux_msg_t *msg)
+{
+    if (!msg) {
+        errno = EINVAL;
+        return false;
+    }
```

Comment:
This function could just return the result of `flux_msg_aux_get()` since that function also returns NULL if msg is NULL.  Also, the explicit setting of errno won't be helpful since the caller won't know whether errno is valid on false return.

Reply:
Yeah, the only reason that is there is to satisfy a unit test. You can test if a function set `errno` by zeroing `errno` first (e.g. as in `strtol(3)`, but I agree no caller would actually do this so will remove.

Reply:
> This function could just return the result of flux_msg_aux_get() since that function also returns NULL if msg is NULL. 

Ah, but I just noticed that we want to return `false` for `msg == NULL` not `true`, so the `if (!msg)` will have to stay, or am I missing something?

Reply:
Sorry, you're right!